src/pages/blog/18-2-release.md

```diff
@@ -0,0 +1,69 @@
+---
+templateKey: blog-post
+title: Ironic 18.2
+author: Julia Kreger
+date: 2021-09-24T16:00:00.000Z
+category:
+  - label: Development Update
+    id: category-A7fnZYrE9
+---
+
+## Xena Release ##
+
+On Wednesday, September 22nd, 2021, the Ironic team released it's OpenStack Project "Xena" cycle deliverable for Ironic as version 18.2.0.
+
+For those who are not aware of how OpenStack branches are supported, this will be a stable branch which will be maintained for a period of time, as opposed to an Intermediate release by the Ironic project which may only see backports for a limited amount of time.
+
+Overall the Xena development cycle consisted of over 22 new features to Ironic, and over 48 thousand lines of code having been modified.
+
+<br>
+
+## Notable Features, Fixes, and Changes ##
+
+This list includes Ironic 18.2, 18.1, and 18.0.
+
+* Parallel image downloads are now enabled by default.
+* Event subscriptions can now be created on Redfish BMC's through the vendor pass-thru interface.
+* *boot_mode* and *secure_boot* are now top level fields of a node which can enable easier management of these states. Some additional work into this should be expected as time goes forward. 
+* Node History events are now recorded, and can be retrieved by the API. Expect CLI tools to be updated to have functionality for this early in the [OpenStack Yoga][https://releases.openstack.org/yoga/schedule.html] development cycle.
+* PXE loaders can now be configured to copy files from the base operating system. You *must* specify this, as Ironic does ship defaults at this time. Look for *[pxe]loader_file_paths* in the configuration.
```

Comment:
You need to insert \ before the [ ] otherwise javascript will try some weird stuff and it will fail =)

*[pxe]loader_file_paths* -> *\[pxe\]loader_file_paths*